include/swift/SIL/MemoryLifetime.h

```diff
@@ -336,6 +350,14 @@ class MemoryDataflow {
         killSet |= loc->subLocations;
       }
     }
+
+    bool exitReachable() const {
+      return exitReachability == ExitReachability::ReachesExit;
+    }
+
+    bool isInInfiniteLoop() const {
```

Comment:
Just for cosmetically, as for the function names, maybe reachesExit() and reachesInfinityLoop() are better